java/pom.xml

```diff
@@ -66,6 +67,13 @@
     <!--        <pact.version>3.5.24</pact.version>-->
     <pact.provider.spring.version>4.0.10</pact.provider.spring.version>
     <pact.consumer.version>4.3.2</pact.consumer.version>
+    <!-- Stacks versions -->
+
+    <!-- Stacks features -->
+    <aws.profile.name>no-aws</aws.profile.name>
```

Comment:
Move "Stacks Features" section up closer to top of properties, place underneath Stacks versions to match other projects - take a look at one of the other pom.xmls in another project